libdedx/dedx.c

```diff
@@ -417,8 +416,29 @@ int _dedx_check_ion(int prog, int ion) {
 	return 0;
 }
 
+float set_conversion_rate(const int old_unit, const int new_unit, const int material, int *err) {
+    const float density = _dedx_read_density(material, err);
+
+    float conversion_rate;
+    if (old_unit == DEDX_MEVCM2G)
```

Comment:
what if old_unit is DEDX_MEVCM2G and new unit is DEDX_KEVUM ?

Then in fact we need to go in two steps:
1. First we go from DEDX_MEVCM2G to DEDX_MEVCM. This means multiplication by density
2. Then we go from DEDX_MEVCM to DEDX_KEVUM. This means multiplication by 0.1

Implementation above does only first step